ci/linkcheck.sh

```diff
@@ -28,4 +28,4 @@ else # running locally
   echo "Checking files changed in $COMMIT_RANGE: $CHANGED_FILES"
 fi
 
-exec mdbook-linkcheck $FLAGS -- $TRAVIS_BUILD_DIR
+exec mdbook-linkcheck $FLAGS -- $GITHUB_WORKSPACE
```

Comment:
Do we still need this?

Reply:
I'm not sure we do. Can we try removing this completely?

Reply:
Okay, will remove it.